test/models/ingridient_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class IngridientTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
We don't want to use standard library provided by rails for testing. Please remove `test` dir from repo, we don't need this.